modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+// TINY-9530
+
+The {productname} package files have been updated with two new icons that enable users to increase or decrease the size of text in the Advanced Code Editor. The new icons are the `text-size-increase` and `text-size-decrease` icons. These new icons have been added to make it easier for users to adjust the size of the text within the code editor while making it easier to read and understand the code they are writing. 
+
+.Example of `textSizeIncrease` and `textSizeDecrease` integration within {productname}.
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+	advcode_inline: true,
+  plugins: [
+    "code", 'advcode', 
+  ],
+  toolbar: "textSizeIncrease textSizeDecrease"
+});
+----
+[NOTE]
+This plugin feature is only available for paid {productname} subscriptions.
+
+=== New xss_sanitization option to allow for XSS sanitization to be disabled
+// TINY-9600
+
+Currently, we use the DOMPurify library to sanitize HTML content when it is inserted into the editor in order to remove unsafe tags & attributes in order to prevent XSS (cross-site scripting). However, customers in some situations have found that the HTML sanitization from the DOMPurify library is changing their content in undesired ways. 
+
+To address this issue, we have added a new `xss_sanitization: boolean` editor option. This option is enabled by default which enables sanitization, but can now be set to `false` to disable sanitization. If a customer wishes to prevent their content from being modified by the sanitization, they can now set this option to `false`.
+
+.Example: using `+xss_sanitization+` to turn DOMPurify off and no longer sanitize HTML against XSS attacks
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  xss_sanitization: false
+});
+----
+
+[IMPORTANT]
+It is important to note that disabling sanitization introduces potential security issues, especially if the server-side XSS sanitization is weak or non-existent and is therefore not recommended.
+
+// Read more about xref:sanitizing-html-input-to-protect-against-xss-attacks[Sanitizing HTML input to protect against XSS attacks]
+
+=== Added top right close 'x' button of the modal dialogs to keyboard navigation
+// TINY-9520
+
+In previous versions of {productname}, the top right close 'x' button within the modal dialogs were not keyboard-navigable by the Tab key.
+
+With the release of {productname} 6.4, various changes have now made it possible for modal dialogues to be navigated with the Tab key. This is accomplished by adding a `Tabbing.config` to the close 'x' button. The `Tabbing.config` is an object that describes the sequence of focusable elements, their `tabindex` and the `initial focus` within the dialog when it is opened. In addition an optional `firstTabstop` property has also been added to the `Tabbing.config` to specify which element receives focus first when the dialog is opened.
```

Comment:
`Tabbing.config` is an Alloy concept, which is not exposed to the integrators.

Reply:
```suggestion
With the latest release of {productname} 6.4, users can now navigate through modal dialogues with ease using the Tab key. This new feature was made possible by updating the tab functions that directly affect the `x` close button, which resulted in changes to the sequence of focusable elements, their `tabindex`, and the `initial` focus within the dialog when it is opened. With these updates, users can now use the `Tab` key to move through the elements within the modal dialogue, making it more accessible and user-friendly. 
```

Reply:
@tiny-ben-tran Can you review the re-write above.

Reply:
```suggestion
With the latest release of {productname} 6.4, users can now navigate through all dialogs that have a `x` close button. With these updates, users can now use the `Tab` key to move through the elements within the modal dialogue, making it more accessible and user-friendly. 
```

Reply:
@tiny-ben-tran ⬆️